Underscore/USConstants.h

```diff
@@ -32,6 +32,7 @@ typedef id   (^UnderscoreReduceBlock)(id memo, id obj);
 typedef void (^UnderscoreArrayIteratorBlock)(id obj);
 typedef id   (^UnderscoreArrayMapBlock)(id obj);
 typedef id   (^UnderscoreArrayZipWithBlock)(id objectFromFirstArray, id objectFromSecondArray);
+typedef void (^UnderscoreIntegerIteratorBlock)(NSInteger n);
```

Comment:
Fair enough, then I think we should assert that `n` is `>= 0` since negative numbers are fundamentally a programmer error here.